### PR TITLE
Added header, footer and some header links components.

### PR DESCRIPTION
**Pull request by sviastyn**
Signed-off-by: sviastyn <svyastynvasul@gmail.com>

### CODE REVIEW

File: src/TechMarket.API/TechMarket.API/Infrastructure/Services/ProductsService.cs [hunk 1]
```diff
@@ -28,6 +29,7 @@ public Product GetById(int id)
         public IEnumerable<Category> GetCategories()
         {
             return this._productsRepository.GetCategories();
-        }
+        }              
+
```
Comment (MorphineForOdin) [inline src/TechMarket.API/TechMarket.API/Infrastructure/Services/ProductsService.cs:33]: A redundant empty line.

File: src/TechMarket.WEB/tech-market/package.json [hunk 2]
```diff
@@ -17,6 +17,7 @@
     "@angular/platform-browser": "~12.0.4",
     "@angular/platform-browser-dynamic": "~12.0.4",
     "@angular/router": "~12.0.4",
+    "bootstrap-icons": "^1.5.0",
```
Comment (MorphineForOdin) [inline src/TechMarket.WEB/tech-market/package.json:20]: Did you manage to integrate icons?

File: src/TechMarket.WEB/tech-market/src/app/app-routing.module.ts [hunk 3]
```diff
@@ -3,7 +3,9 @@ import { RouterModule, Routes } from '@angular/router';
 
 const routes: Routes = [
     { path: 'login', loadChildren: () => import('./pages/login/login.module').then(m => m.LoginModule) },
-    { path: '', loadChildren: () => import('./pages/landing/landing.module').then(m => m.LandingModule) }
+    { path: '', loadChildren: () => import('./pages/landing/landing.module').then(m => m.LandingModule) },
+    { path: 'products', loadChildren: () => import('./pages/products/products.module').then(m => m.ProductsModule) },
```
Comment (MorphineForOdin) [inline src/TechMarket.WEB/tech-market/src/app/app-routing.module.ts:7]: The empty route always should be the last in the routes array.

File: src/TechMarket.WEB/tech-market/src/app/app.component.html [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
+<header class="header-position">
     <tm-header></tm-header>
 </header>
 
 <main>
     <router-outlet></router-outlet>
 </main>
 
-<footer>
+<footer class="footer-position">
```
Comment (MorphineForOdin) [inline src/TechMarket.WEB/tech-market/src/app/app.component.html:9]: You don't need CSS classes for styling header and footer.

File: src/TechMarket.WEB/tech-market/src/app/app.module.ts [hunk 5]
```diff
@@ -6,18 +6,22 @@ import { AppComponent } from './app.component';
 import { CoreModule } from './core/core.module';
 import { LandingModule } from './pages/landing/landing.module';
 import { LoginModule } from './pages/login/login.module';
+import { ProductsModule } from './pages/products/products.module';
+import { CartModule } from './pages/cart/cart.module';
 
 @NgModule({
     declarations: [
-        AppComponent
+        AppComponent,
```
Comment (MorphineForOdin) [inline src/TechMarket.WEB/tech-market/src/app/app.module.ts:14]: Remove trailing coma

File: src/TechMarket.WEB/tech-market/src/app/core/footer/footer.component.html [hunk 6]
```diff
@@ -1 +1,30 @@
...
+            </li>
+            <li>
+                <a href="mailto:techmarket@mail.com">techmarket@mail.com</a>
+            </li>
+        </ul>
+    </div>
+
+    <div class="schedule-container">
+        <ul>
+            <p>Work schedule</p>
```
Comment (MorphineForOdin) [inline src/TechMarket.WEB/tech-market/src/app/core/footer/footer.component.html:25]: It is an online store - so we shouldn't have a work schedule.

File: src/TechMarket.WEB/tech-market/src/app/core/header/header.component.html [hunk 7]
```diff
@@ -1 +1,27 @@
-<div>Header would be implemented soon...</div>
+<div class="header-container">
+
```
Comment (MorphineForOdin) [inline src/TechMarket.WEB/tech-market/src/app/core/header/header.component.html:2]: A redundant empty line.

File: src/TechMarket.WEB/tech-market/src/app/core/header/header.component.html [hunk 8]
```diff
@@ -1 +1,27 @@
...
+    <div class="header-mobile">
```
Comment (MorphineForOdin) [inline src/TechMarket.WEB/tech-market/src/app/core/header/header.component.html:3]: header-mobile?

File: src/TechMarket.WEB/tech-market/src/app/core/header/header.component.html [hunk 9]
```diff
@@ -1 +1,27 @@
...
+        <div class="logo">
+            <a href="http://localhost:4200">Tech Market</a>
```
Comment (MorphineForOdin) [inline src/TechMarket.WEB/tech-market/src/app/core/header/header.component.html:5]: Please, use relative URLs.

File: src/TechMarket.WEB/tech-market/src/app/core/header/header.component.html [hunk 10]
```diff
@@ -1 +1,27 @@
...
+        <button class="burger-button" (click)="burgerClick()">
+            <i>=</i>
+        </button>
+    </div>
+
+    <div *ngIf="menuOpened" class="header-links">
+        <div class="nav-links">
+            <a href="http://localhost:4200/products">Products</a>
+            <a>Search</a>
+            <button (click)="testAuth()">AuthTest</button> <!--DELETE this row-->
```
Comment (MorphineForOdin) [inline src/TechMarket.WEB/tech-market/src/app/core/header/header.component.html:17]: Delete this row.

File: src/TechMarket.WEB/tech-market/src/app/pages/landing/landing.component.html [hunk 11]
```diff
@@ -5,5 +5,6 @@
     <ul *ngFor="let user of users">
         <li>{{user.firstName}} </li>
         <li>{{user.lastName}}</li>
+        <li>{{user.email}}</li>
```
Comment (MorphineForOdin) [inline src/TechMarket.WEB/tech-market/src/app/pages/landing/landing.component.html:8]: Please, revert this change.

File: src/TechMarket.WEB/tech-market/src/app/pages/login/login.component.html [hunk 12]
```diff
@@ -1 +1,16 @@
...
+<div class="login-form">
+    <form>
+        <ul>
+            <p class="login-form-title">Personal account</p>
+            <li>
+                <label for="mail"></label>
+                <input type="text" placeholder="Enter E-mail" name="mail" required>
+            </li>
+            <li>
+                <label for="psw"></label>
```
Comment (MorphineForOdin) [inline src/TechMarket.WEB/tech-market/src/app/pages/login/login.component.html:10]: password

**Review by MorphineForOdin — CHANGES_REQUESTED** [review]

File: src/TechMarket.WEB/tech-market/package.json (hunk 2 above)
Comment (sviastyn) [reply src/TechMarket.WEB/tech-market/package.json:20]: I decided to download free svg icons, and move them into assets.

**Review by MorphineForOdin — APPROVED** [review]

**Pull request merged**